pkg/controllers/proxy/network_services_controller.go

```diff
@@ -1059,9 +1100,9 @@ func (nsc *NetworkServicesController) syncIpvsServices(serviceInfoMap serviceInf
 	// cleanup stale IPs on dummy interface
 	glog.V(1).Info("Cleaning up if any, old service IPs on dummy interface")
 	addrActive := make(map[string]bool)
-	for k, endpoints := range activeServiceEndpointMap {
+	for k := range activeServiceEndpointMap {
```

Comment:
@murali-reddy The problem I had is that services that does not have any endpoints are flapping, making many unnecessary LN calls to add and remove the service on every sync. also if the only endpoint in the service is gracefully terminating we can't yank the service out under it.